docs/Design/SoftDetailedDes/MIS.tex

```diff
@@ -630,6 +630,150 @@ \subsubsection{Local Functions}
 
 \newpage
 
+\section{MIS of Users Module} \label{mUsers} 
+The Users Module exists to handle CRUD operations for operations related to users and their grocery trips.
+
+\subsection{Module}
+
+users
+
+\subsection{Uses}
+
+ Database Driver Module (Section \ref{mDBDriver}) 
+
+\subsection{Syntax}
+
+\subsubsection{Exported Constants}
+\textbf{Item} as defined in Section \ref{mExtraction} \\
+
+\noindent \textbf{GroceryTrip} as defined in Section \ref{mExtraction} \\
+
+\noindent \textbf{User}
+
+\begin{table}[H]
+  \begin{tabular}{|p{0.25\textwidth}|p{0.25\textwidth}|p{0.40\textwidth}|}
+    \hline
+    \textbf{Output Name} & \textbf{Output Type} & \textbf{Description} \\
+    \hline
+    userId & Integer & Unique user id number \\
```

Comment:
I thought userId was a String? Or at least that's what I put in my tables. I thought String was easier to work with and it's what I always worked with in my past co-ops for IDs. Otherwise can you change the data type for me to align with this (Receipt Extraction Module)?

Reply:
typically db primary keys are integer Ids pretty sure 

but we can change it later ... ill make it a string for now

Reply:
Think maybe you forgot or it didn't merge properly, but it's still showing up as an Integer.